hieradata_aws/integration.yaml

```diff
@@ -10,6 +10,7 @@ cron::daily_hour: 6
 
 environment_ip_prefix: '10.1'
 
+govuk::apps::account_api::plek_account_manager_uri: 'https://www.account.staging.publishing.service.gov.uk'
```

Comment:
Should this be staging or integration?

Reply:
```suggestion
govuk::apps::account_api::plek_account_manager_uri: 'https://www.account.integration.publishing.service.gov.uk'
```

Reply:
This should actually be staging, as we're not running a separate integration account manager - see also finder-frontend's hieradata

Reply:
ok ace, will approve